theme/blast_report.tpl.php

```diff
@@ -62,273 +62,264 @@
 
```

Comment:
Except for handling of the new max_results variable, changes in this file are all re-formatting, to make it easier to see the embedded HTML elements.